crates/etl/src/lib.rs

```diff
@@ -173,6 +173,7 @@ where
 type HeapItem = (Reverse<(Vec<u8>, Vec<u8>)>, usize);
 
 /// `EtlIter` is an iterator for traversing through sorted key-value pairs in a collection of ETL
+///
```

Comment:
I would say that it is not necessarily a good idea to downgrade our clippy on this. Since these are new suggestions related to an update, I think it would be better to apply them everywhere instead to always stay aligned with what the clippy offers.

Reply:
If this is only in one place, you can probably put an `#[allow(...)]` on the line just before this statement to allow this not to warn only for this.